Shared/Chem_Shared/Lightning_mod.F90

```diff
@@ -1517,20 +1573,21 @@ subroutine LOPEZ_FlashRate(IM, JM, LM, FRLAND, ZKBCON, CAPE,  &
     real,    dimension(1:lm) :: q_graup,q_snow,rho
 
 
+!!! DEBUGGING
+!    print*, ""
```

Comment:
This might be a good place to use pFlogger if you'd like to keep these around. @tclune or myself (or @JulesKouatchou, but he's on leave for a while) could help with this.

Reply:
Sure, if there's a better way to do that, I will.  Is there a wiki, or example in GEOS?  I'm not familiar with pFlogger.

Reply:
@JulesKouatchou has been working on a [wiki for pFlogger directly](https://github.com/Goddard-Fortran-Ecosystem/pFlogger/wiki/pFlogger_UserGuide), but it's still in draft and more "general". We were looking for a good place for him to work on a place to test and document its use in GEOS.

A good place to look at it would be [the sun code in MAPL](https://github.com/GEOS-ESM/MAPL/blob/main/base/MAPL_sun_uc.F90) which is pretty simple. If you search for `lgr` you can get a flavor of it a la:
```fortran
  use pflogger, only: logging, Logger
...
      class(Logger), pointer :: lgr
      lgr => logging%get_logger('MAPL.SUN')
...
            call lgr%debug('  tsi at end of table: %F8.3', tsi(numlines))
...
            call lgr%debug('Off the end of table, moving into last complete cycle')
            call lgr%debug('  Original Year-Mon-Day to Find: %i0.4~-%i0.2~-%i0.2', originalYear,originalMon,originalDay)
...
```

So you can see things like `%f8.3` is a float in Fortran style, same for `%i` and `%a`

Now, the interesting part which we might need @tclune's help on is the `get_logger` statement. Technically, I think the logger for Chem GC would be `CAP.GCM.AGCM.PHYSICS.CHEMISTRY`. But I'm not sure what the "right" way to it is to delineate it in `Shared/Chem_Shared/Lightning_mod.F90`. `CAP.GCM.AGCM.PHYSICS.CHEMISTRY.SHARED.LIGHTNING`?

I suppose maybe we can get your code in now, and then when @JulesKouatchou returns, we can maybe use this component as a first example of using pFlogger in GEOS